lib/novu/http.ex

```diff
@@ -53,7 +53,7 @@ defmodule Novu.Http do
   Makes a `POST` request to Novu.
   """
   @spec post(url :: Req.url(), body :: map()) :: response()
-  def post(url, body) do
+  def post(url, body \\ %{}) do
```

Comment:
Since empty post bodies are less common and could potentially cause issues, what do you think about remove the default and being explicit about the empty body in the places we need to?

Reply:
Sounds good, I'll fix it